systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PodUtils.java

```diff
@@ -128,9 +134,10 @@ public static String getFirstPodNameContaining(String searchTerm) {
     }
 
     public static void waitForPod(String name) {
+        resourceOperationTimeout = Constants.TIMEOUT_FOR_KAFKA_ROLLER_BACKOFF;
```

Comment:
You should probably rename `Constants.TIMEOUT_FOR_KAFKA_ROLLER_BACKOFF` if you want to use it for any kind of pods